yaml/vendor.conf

```diff
@@ -0,0 +1,34 @@
+github.com/Azure/go-ansiterm 388960b655244e76e24c75f48631564eaefade62
+github.com/Microsoft/go-winio fff283ad5116362ca252298cfc9b95828956d85d
+github.com/Nvveen/Gotty cd527374f1e5bff4938207604a14f2e38a9cf512
+github.com/Sirupsen/logrus v0.11.0
+github.com/agl/ed25519 5312a61534124124185d41f09206b9fef1d88403
+github.com/docker/distribution 545102ea07aa9796f189d82f606b7c27d7aa3ed3
+github.com/docker/docker 4163b5cbd59e2a5dee411310cb49734472f95440
```

Comment:
I may be confused, but we're vendoring docker/docker here, and create a custom client / generator using the _vendored_ code; as a result, the output of the commands / flags will never automatically update to match the current version of the source code?